protos/flow.proto

```diff
@@ -302,7 +302,7 @@ message QRepParitionResult {
 }
 
 message DropFlowInput {
-  string flow_name = 1;
+  string flow_job_name = 1;
```

Comment:
renaming fields breaks things

Reply:
proto wasn't used before this PR for some reason